CorsixTH/Src/th_map.cpp

```diff
@@ -645,6 +629,26 @@ void THMap::setParcelOwner(int iParcelId, int iOwner)
     _updatePurchaseMatrix();
 }
 
+void THMap::_checkDividers(THMapNode* pNode, const THMapNode* pOriginalNode,
```

Comment:
Okay. How should the method get the THMap object to call? Should it also be a part of the argument list?